### PR TITLE
Update to support sass-loader 8.x

### LINKED ISSUE #2206: Compatibility with sass-loader 8.0.0
-   Laravel Mix Version: 4.1.2

### Description:

A heads up that Laravel mix doesn't seem to be compatible with sass-loader 8.0.0 (just released) which has some breaking changes.

### Steps To Reproduce:

See build failure in https://github.com/danielbachhuber/simple-event-tracking/pull/3

### PR DESCRIPTION
**Pull request by dshoreman**
Wraps the options for Dart/Node-sass in a `sassOptions` object and updates versions of certain packages according to [requirements for sass-loader 8.0.0](https://github.com/webpack-contrib/sass-loader/blob/master/CHANGELOG.md).

Fixes #2206

### CODE REVIEW

**Comment by Nothing-Works** [comment]
@dshoreman any idea why this is still failing?

**Comment by dshoreman** [comment]
@Nothing-Works 	I can't say for sure, but after checking the recent commits it seems AppVeyor's been failing since eslint was added 3 commits back in 501a6d12.

The site isn't loading for me at the moment so I can't check the exact error, but IIRC it was complaining about not being able to find 'node'. Looking at the .eslintrc in the above commit it seems to be referring to line 7 (`"node": true`), but what it *should* be I don't know.

tl;dr: AppVeyor isn't even running because eslint config seems broken, but Travis is passing :tada:

**Comment by HDVinnie** [comment]
When will a Mix update be pushed that includes this fix?

**Comment by DannyFeliz** [comment]
> When will a Mix update be pushed that includes this fix?

@HDVinnie Done [v5.0.0](https://github.com/JeffreyWay/laravel-mix/releases)
